.gitignore

```diff
@@ -25,4 +25,6 @@ secret_key
 config.py
 
 # Generated images for the honeypots (big binary blobs)
-honeypot_images/*.img
\ No newline at end of file
+honeypot_images/*.img
+/.idea/misc.xml
```

Comment:
Three things I'd like to get clarification on for the .gitignore change:
- Why add these?
- Why the leading / ?
- They should be with the other project specific ignores

Reply:
Those changes crept in due to my development environment and don't seem essential. I've removed them in latest commit.